src/components/Datagrid/renderers.tsx

```diff
@@ -78,16 +80,14 @@ export const StandardRenderers: Record<string, CellValueRenderer> = {
 };
 
 export function renderMonetaryAmount(
-  value: number,
+  value: BigNumber | number,
   currency: string,
-): ReactNode {
+) {
+  const val = BigNumber(value);
   if (currency === "USD") {
-    return `$${value.toLocaleString("en", {
-      minimumFractionDigits: 2,
-      maximumFractionDigits: 2,
-    })}`;
```

Comment:
Do we not still want to always show two decimal places for dollar values?

I.e. for 11.5 show $11.50?

Perhaps it's worth considering using the [browser's built-in currency formatting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#currencydisplay)?